Library/Homebrew/cmd/help.rb

```diff
@@ -89,4 +93,15 @@ def command_help(path)
       end.join.strip
     end
   end
+
+  def command_help_cmd(path)
+    path_str = path.to_s
+    cmd = path_str.match(%r{Homebrew/cmd/(.*).rb})[1]
+    class_name = "#{cmd.capitalize}Command"
+    class_instance = Object.const_get("Homebrew::#{class_name}").new
+    <<-EOS.undent
+      brew #{cmd} [#{class_instance.valid_options.keys.join "]["}]
```

Comment:
It might be nice if this could be moved into a return value somewhere so as to give the reader a little more understanding of what's going on. Something like `class_instance.valid_option_names.join("] [")`?